ipvs.py

```diff
@@ -318,6 +325,22 @@ def to_dict(self):
                 'sched': self.sched_,
             }
 
+    def to_attr_list(self):
+        if self.fwmark_ is None:
+            af, addr = _to_af_union(self.vip_)
+            netmask = ((1 << 32) - 1) if af == socket.AF_INET else 128
+            proto = self.proto_num()
+            return IpvsServiceAttrList(af=af, addr=addr, protocol=proto,
+                                       netmask=netmask, port=self.port_,
+                                       sched_name=self.sched_,
+                                       flags=struct.pack(str('=II'), 0, 0))
+        else:
+            af = socket.AF_INET
```

Comment:
As per https://github.com/facebook/gnlpy/blob/master/ipvs.py#L418, fwmark creation required an AF. We will work on exposing the address family in Service object so you can update this part of your diff